connect_ext_test.go

```diff
@@ -2079,6 +2079,46 @@ func TestHandlerReturnsNilResponse(t *testing.T) {
 	assert.Equal(t, panics, 2)
 }
 
+func TestStreamUnexpectedEOF(t *testing.T) {
+	t.Parallel()
+	mux := http.NewServeMux()
+	mux.HandleFunc("/connect.ping.v1.PingService/CountUp", func(responseWriter http.ResponseWriter, request *http.Request) {
+		io.Copy(io.Discard, request.Body) //nolint:errcheck
+		header := responseWriter.Header()
+		header["content-type"] = []string{"application/connect+json"}
```

Comment:
nit: Doesn't practically matter, but this is cleaner in my opinion.
```suggestion
		header.Set("Content-Type", "application/connect+json")
```